app/helpers/hosts_helper.rb

```diff
@@ -270,6 +276,16 @@ def overview_fields(host)
     fields
   end
 
+  def host_detailed_status_list(host, options = {})
+    content_tag(:ul, {:id => 'host_status_list'}.merge(options)) do
+      host.host_statuses.map do |status|
+        content_tag(:li,
+                    content_tag(:i, '&nbsp'.html_safe, :class => host_global_status_icon_class(status.to_global)) +
+                      content_tag(:span, status.to_label, :class => host_global_status_class(status.to_global), :title => _(status.name)))
```

Comment:
I'm also not terribly keen on only having the name in a tooltip, and along with the comment about about whether it should be expanded by default, would prefer something like a table:

Host status: OK
Build: Installed
Configuration: Something